Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
       highline (~> 2.0.0)
-    database_cleaner (1.7.0)
-    diff-lcs (1.3)
+    database_cleaner (1.8.5)
+    diff-lcs (1.4.2)
     erubi (1.9.0)
     event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
     highline (2.0.3)
-    method_source (0.9.2)
-    mustermann (1.0.3)
+    method_source (1.0.0)
+    mustermann (1.1.1)
```

Comment:
### [mustermann](https://github.com/sinatra/mustermann)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.3 → 1.1.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **29** commits. See the full changes on [the compare page](https://github.com/sinatra/mustermann/compare/v1.0.3...v1.1.1).

These are the first 10 commits:
- (8505609) [fix broken link on README [ci skip]](https://github.com/sinatra/mustermann/commit/85056093682e61f18857048122d773f7ee1fda8d)
- (6310751) [don't use adjusted values for reporting error](https://github.com/sinatra/mustermann/commit/63107511551d36519f5d6dce142f130ea88f7c38)
- (54ed0b6) [Merge pull request #92 from namusyaka/fix-88](https://github.com/sinatra/mustermann/commit/54ed0b63f326ee9c8eb3f20d0aacccb7fd4a62f6)
- (635b068) [upgrade ruby versions](https://github.com/sinatra/mustermann/commit/635b06872f46351006c2c91733504b1086af277c)
- (a05080b) [Merge pull request #93 from sinatra/travis](https://github.com/sinatra/mustermann/commit/a05080b2c776f61e8df38a0fb51f35439bd64133)
- (5625972) [upgrade from 2.2.9 to 2.2.10](https://github.com/sinatra/mustermann/commit/5625972edfddb3d254acd949a1a312cca5167de3)
- (0753081) [fix rvm test](https://github.com/sinatra/mustermann/commit/0753081d7049218f41d72fb57cd61a989a4a6c85)
- (07cc283) [Merge pull request #96 from horaciob/change_ci_config](https://github.com/sinatra/mustermann/commit/07cc283baee9ec4f894662fcab808663f8da96b3)
- (e727662) [Use Hash#merge explicitly instead of `**` operators](https://github.com/sinatra/mustermann/commit/e727662a330710ea0c0b54c49acdccf0b37c37f9)
- (57ae96c) [Make it explicit the optimization condition for pattern gene…](https://github.com/sinatra/mustermann/commit/57ae96cee7fc1b9700929143c9f107053803cc18)


</details>